Book-A-Desk.Common/Book-A-Desk.Common.fsproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Suggestion from Robert: rename project to Core, make it clear where it fits in the architecture.